server/controllers/article.controller.js

```diff
@@ -28,6 +29,7 @@ class ArticleController {
       title,
       body,
       tags,
+      image: [image],
```

Comment:
Good work, in cases where there are no images in the article, what would happen to this input?

Reply:
It returns null